notebooks/binary_predicates.ipynb

```diff
@@ -0,0 +1,1157 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        """Times the speed of producing the test data as well as the predicate execution.
</pre><p>Why should you time the data production? Benchmark should only time the kernel.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>